tools/StaticAnalysis/Exceptions/Az.EventHub/BreakingChangeIssues.csv

```diff
@@ -0,0 +1,2 @@
+AssemblyFileName,ClassName,Target,Severity,ProblemId,Description,Remediation
```

Comment:
Please keep the double quotes, refer to https://github.com/Azure/azure-powershell/blob/master/tools/StaticAnalysis/Exceptions/Az.Automation/BreakingChangeIssues.csv

Reply:
Thank you, the file is modified to retain double quotes.